java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -63,12 +89,18 @@ public interface ServerManagementClusterCommands {
      * @param options A list of {@link InfoOptions.Section} values specifying which sections of
      *     information to retrieve. When no parameter is provided, the {@link
      *     InfoOptions.Section#DEFAULT} option is assumed.
-     * @param route Routing configuration for the command. Client will route the command to the nodes
-     *     defined.
+     * @param route Specifies the routing configuration for the command. The client will route the
+     *     command to the defined nodes.
      * @return Response from Redis cluster with a <code>String</code> with the requested sections.
      *     When specifying a <code>route</code> other than a single node, it returns a <code>
      *     Map{@literal <String, String>}</code> with each address as the key and its corresponding
      *     value is the information of the sections requested for the node.
+     * @example
+     *     <pre>{@code
+     * ClusterValue<String> payload = clusterClient.info(InfoOptions.builder().section(STATS).build(), RANDOM).get();
+     * // Command is routed to a single, randomly chosen node, expecting a SingleValue result.
```

Comment:
Command is routed to a single, randomly chosen node, because RANDOM was passed as route